autogen/main/moved.tf.tmpl

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# Fix the name typo in the previous ConfigMap creation call
+moved {
+  from = kubernetes_config_map_v1_data.kube-dns-upstream-namservers
+  to   = kubernetes_config_map_v1_data.kube-dns-upstream-nameservers
```

Comment:
I wonder if this should be direct to `kube_dns_upstream_nameservers`?  No sure if they support "chaining"?

Also very likely we won't have a release with the `kube-dns-upstream-nameservers` anyway.

Reply:
Fair. I'd assumed it would work since the "from" is different, so it would just relocate either that was found?

If you're sure they're going out together, I'll update it now to just have the one. 👍 
cc: @MaximF

Reply:
What I don't know is if the move rules are "chained", if they are not and this rule fires, it will just try to move to `kube-dns-upstream-nameservers`, which of course no longer exists.  As this PR also adds `kube_dns_upstream_nameservers`, regardless it should be perfectly safe to go directly to that resource.